Sources/XiEditor/LineCache.swift

```diff
@@ -60,6 +54,38 @@ struct Line<T> {
     }
 }
 
+enum AnnotationType: String {
+    case Selection = "selection"
+    case Find = "find"
+}
+
+extension AnnotationType {
+    static var all: [AnnotationType] {
```

Comment:
We should be able to get this from the stdlib; check out https://docs.swift.org/swift-book/LanguageGuide/Enumerations.html#ID581